src/Plugins/ResizeMirror/tests/ResizeMirror.test.js

```diff
@@ -0,0 +1,139 @@
+import {
+  createSandbox,
+  waitForRequestAnimationFrame,
+  clickMouse,
+  moveMouse,
+  releaseMouse,
+  waitForDragDelay,
+  DRAG_DELAY,
+} from 'helper';
+import {Draggable} from '../../../..';
+import ResizeMirror from '..';
+
+const sampleMarkup = `
+  <ul class="Container">
+    <li>Smaller item</li>
+  </ul>
+  <div class="Container">
```

Comment:
Kinda wacky sample markup? Even though its a test... I can't help but feel like we either don't use `<li>` or we change the `div` to `ul`

Reply:
oups, that was definitely not intended... no idea what I was thinking, will change